sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -477,6 +478,10 @@ internal class RadarApiClient(
             queryParams.append("&groups=${groups.joinToString(separator = ",")}")
         }
 
+        chainMetadata?.entries?.forEach {
+            queryParams.append("&chainMetadata%5B${it.key}%5D=%22${it.value}%22");
```

Comment:
Here's how `searchGeofences` is doing it, we should probably try to keep `searchGeofences` and `searchPlaces` consistent (either both encoded or unencoded):

https://github.com/radarlabs/radar-sdk-android/blob/e27b44c48bacc7e551f4d8f154a908e0b978a0b0/sdk/src/main/java/io/radar/sdk/RadarApiClient.kt#L534

Reply:
I think that the value has to be in quotations, so I can do `&chainMetadata[${it.key}]=\"${it.value}\""`. The escapes were copied from iOS, which has to escape the quotations because you can't put literal quotations in a string literal.